pkg/gameserverallocations/controller_test.go

```diff
@@ -1161,6 +1162,8 @@ func TestMultiClusterAllocationFromRemote(t *testing.T) {
 
 func TestCreateRestClientError(t *testing.T) {
 	t.Parallel()
+	err := runtime.ParseFeatures("")
```

Comment:
Can you explain why this needed to be added?

Reply:
This test relies on createRemoteClusterDialOption which now relies on feature flags being enabled.

Reply:
@markmandel - it seems like we shouldn't have to call ParseFeatures with an empty list in test code; shouldn't the default behavior when checking for a feature be to use the compiled in list (e.g. we should only have to call this in tests where we are using a non-default feature flag)?

Reply:
The reason I needed to do this was because the key was originally MTLSEnabled, which should be true by default. But if the key wasn't in the map, the default value returned is false, which means that all of the logic that was being test was bypassed, since we would immediately return grpc.WithInsecure. By changing the feature semantics to MTLSDisabled, the default value returned by the map and the expected default value match now.

Reply:
Marking this as resolved since this is no longer an issue when using a helm configuration parameter.